### PR TITLE
Adds Semantic versioning - github releases

### PR DESCRIPTION
**Pull request by robertke**
Need to find a way of using different library instead of installing on server SetuptoolsVersion pkg_resources import SetuptoolsVersion external library

### CODE REVIEW

File: library/delete_old_deployments.py [hunk 1]
```diff
@@ -26,12 +26,33 @@
...
     return out
 
+def get_git_hash_dir(module, deployment_dir):
+    command = "find . -maxdepth 1 -type d -exec printf '{}\n' \; | awk 'length==42' | sed 's|./||'"
+    hash_dirs = exec_command(module, command, deployment_dir)
+    hash_dirs = hash_dirs.split("\n")
+    return hash_dirs
+
+def get_sem_ver_dir(module, deployment_dir):
+    command = "find . -maxdepth 1 -type d -exec printf '{}\n' \; | sed 's/.\|.\///'"
```
Comment (kieranajp) [inline library/delete_old_deployments.py]: `find -maxdepth 1 -type d | sort | grep '[0-9+]\.[0-9+]\.[0-9+]'` should do it without the need for a semver library.

@ahelal @robertke

Comment (ahelal) [inline library/delete_old_deployments.py]: @kieranajp that will not work for *1.2.0-rc.1* for example for more info http://semver.org/

Comment (kieranajp) [inline library/delete_old_deployments.py]: It does, actually. Tested with that before posting here. Grep regex is weird.

Comment (ahelal) [inline library/delete_old_deployments.py]: @kieranajp since you guys are the only users. just push your changes and we can try it.

**Comment by robertke** [comment]
@ahelal can you look into this PR please, i think it's ready to merge

**Comment by ahelal** [comment]
+1

**Pull request merged**